safe_transaction_service/history/indexers/tx_processor.py

```diff
@@ -235,9 +237,10 @@ def swap_owner(
         MultisigConfirmation.objects.remove_unused_confirmations(
             contract_address, safe_status.nonce, owner
         )
+        safe_message_models.SafeMessageConfirmation.objects.filter(owner=owner).delete()
 
     def store_new_safe_status(
-        self, safe_last_status: SafeStatus, internal_tx: InternalTx
+        self, safe_last_status: SafeLastStatus, internal_tx: InternalTx
```

Comment:
What is the reason to change the type here?

Reply:
It was a typo before and I detected it when checking the `tx_processor` for implementing this feature